aries-site/src/examples/templates/wizard/components/CancellationLayer.js

```diff
@@ -16,9 +16,7 @@ export const CancellationLayer = ({ onSetOpen, ...rest }) => {
     >
       <Box pad="large" gap="medium" width="large">
         <>
-          <Heading color="text-strong" margin="none">
-            Cancel
-          </Heading>
+          <Heading margin="none">Cancel</Heading>
```

Comment:
```suggestion
          <Heading level={2} margin="none">Cancel</Heading>
```
Since this is a Layer.